src/GslibIO.jl

```diff
@@ -43,6 +51,38 @@ function load(file::File{format"GSLIB"})
     georef(data, domain)
   end
 end
+"""
+    parse_gslib(filename::AbstractString)
+
+Helper function to parse a standard GSLIB file.
+It returns the data as a matrix `X`, and `metadata` that includes the parsed information.
+"""
+
+function parse_gslib(filename::AbstractString)
+  open(filename) do fs
+    # header
+    header = readline(fs)
+    # number of variables is the first integer in the second line (it may contain extra info)
```

Comment:
this comment is not very helpful either. this type of information should live in the struct definition. here you just need to read the spec following the name conventions adopted in the struct and people will understand clearly what is happening.

Reply:
this comment has not been addressed, yet you marked it as resolved